### ISSUE TITLE
General Definition not visible: GamePhysics

### ISSUE THREAD

**Issue by oluowoj** [JacquesCarette/Drasil#1372]
Looks like there was an issue with this piece and it was turned off, I will investigate. The code in Drasil needs a lot of improvement and there are currently no derivations for any of the definitions

**Comment by Mornix** [comment]
Something to note: this difficulty may be a symptom of GamePhysics using `Assembler` and while the rest of the DocLang and Drasil have made improvements, they haven't been exposed through the GamePhysics-specific (or at least specifically used) `Assembler`.

**Comment by oluowoj** [comment]
@Mornix, what is the best way to approach this issue? Should I plug in the file to see what's going on first?

**Comment by Mornix** [comment]
Can't hurt trying to plug in the file. There is an outstanding issue (#927) to look at what parts of `Assembler` are good and worth integrating into the rest of the examples.

**Comment by oluowoj** [comment]
@JacquesCarette , @Mornix, checking if you have any clue on this. I plugged in Gen. Definition  file for GamePhysics , renamed the file from GDefs to GenDefs.hs for consistency with other examples and rewritting  module. I am getting this error, although the Module name and file name are thesame.  Found a few similar errors online but no resolution yet.

<img width="673" alt="image" src="https://user-images.githubusercontent.com/43192745/62655359-cd458e80-b92f-11e9-9781-e5dbb489c1aa.png">
